app/human-detector-app/package.json

```diff
@@ -38,7 +38,7 @@
     "expo-device": "~4.2.0",
     "expo-font": "~10.1.0",
     "expo-linking": "~3.1.0",
-    "expo-modules-core": "^0.11.8",
+    "expo-modules-core": "~0.9.2",
```

Comment:
Why did `expo-modules-core` have to changed here?

Reply:
Figured I would sneak a fix in with this PR, I pulled the wrong version in #39 and Expo was complaining about it